modern-humanities-research-association.csl

```diff
@@ -102,16 +102,13 @@
       </else>
     </choose>
   </macro>
-  <macro name="disambiguate">
+  <macro name="title-subsequent">
     <choose>
       <if disambiguate="true">
         <choose>
-          <if variable="title" match="none">
```

Comment:
I think you want to leave that test for rare cases (like a letter without a title) but you want to remove the disambiguate="true" test above)